10-05/task_2/task2.rb

```diff
@@ -0,0 +1,13 @@
+parag_counter = 0
```

Comment:
All this code can be replaced with a one-liner:
```ruby
parag_counter = File.readlines("parag1.txt").count { |line| line =~ /\S+\n$/ } # \S means non-empty symbol (word, number, or dot etc)
puts parag_counter
```

Reply:
<img width="957" alt="Screenshot 2020-05-25 at 12 27 15" src="https://user-images.githubusercontent.com/64412150/82799739-2e206200-9e83-11ea-8425-fe0cc2153c82.png">
here I used pry debugger to quickly check different approaches and find the best solution